panel/src/components/Dialogs/SearchDialog.vue

```diff
@@ -105,6 +104,14 @@ export default {
 				this.$panel.searches[this.type] ??
 				Object.values(this.$panel.searches)[0]
 			);
+		},
+		type() {
+			const type = this.$panel.view.search;
+
+			return (
+				this.$panel.searches[type] ?
+					type : Object.keys(this.$panel.searches)[0]
+			);
```

Comment:
Won't this break the types dropdown if you move `type` from data to be a computed prop? cause now the type cannot be reassigned.

Reply:
I don't know the Vue process very well. Data and computed do the same thing in my knowlegde/view 🙈 One is predefined and the other is computed 😅

Reply:
Yes, but if it is a computed prop, you cannot do `type = 'users'` anymore. But that's what's done via the type dropdown.